### DISCUSSION TITLE
Can't apply for GitHub Student Developer Pack and no support ticket is not being responded to

### DISCUSSION THREAD

**Discussion by wasay-k** [community/community#87202] [GitHub Education]
### Hi everyone,

I tried applying for the Student Developer Pack, but I keep getting the following error:
```
The school you selected does not appear to have a campus location in your country. Your school-provided academic affiliation document must indicate virtual learning if you are remote. If you believe that the school you have applied for has incorrect information that affects your current application e.g. incorrect email domain, please select ‘My selected school has incorrect or incomplete information e.g. domains or campus location’ option when submitting a GitHub Education support ticket.

You appear not to be near any campus location for the school you have selected. Your school-provided academic affiliation document must indicate virtual learning if you are remote.Please enable the application to discover your location from the browser pop-up to provide an accurate location. If you are using a VPN, please disable it when you reapply. If you believe that the school you have applied for has incorrect information that affects your current application e.g. incorrect campus location, please select ‘My selected school has incorrect or incomplete information e.g. domains or campus location’ option when submitting a GitHub Education support ticket. 
```

The school I study in is in the same city that I live in, but I keep getting this error. I submitted a support ticket over a week ago, and it has still not been responded to. Is there something else I can do? Please do tell if there is another way.

**Comment by Valeriucoding** [comment]
I'm encountering the same issue too, did you manage to do anything?

**Reply by wasay-k** [reply]
Unfortunately, no. Is there a way we could contact GitHub elsewhere?

**Reply by mishmanners** [reply]
Do you have a VPN turned on that could be changing the location of your IP address?

**Reply by wasay-k** [reply]
No, I don't.

**Reply by mishmanners** [reply]
Okay, can you try sending in an application when you're on campus and connected to the campus network?

**Comment by barart** [comment]
Have the same problem, i cant find a way to open a ticket, i already applied and approved in past but now i have this problem too, im in camp , do you find a way to solve this?
